### ISSUE TITLE
There is no functionality provided to adjust columns width which is achievable using single pointer.:A11y_.NetAspireDashboard_Resources_Detailsview_DraggingMovement

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3365]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Tags:
#A11yTCS; #A11yMAS; #A11ySev3; #A11yWCAG2.2; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #DraggingMovement; #WCAG2.5.7;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

**Repro Steps:**
1. Hit the URL: https://localhost:15887 and login with valid credentials.
2. TAB to "Resources" tab item and hit ENTER key.
3. TAB to "View" button present under 'Details' column header in the page and hit ENTER key.
4. "Details container" will open.
5. Details section is adjustable using dragging, observe if functionality is provided to adjust the details view section using single pointer.

### Actual Results:
There is no functionality provided to adjust the height of "Details View" section which is achievable using single pointer.

**Note:
Issue 1: Same issue reproes on 'Structured Logs' page -> Table->Details-> View and Traces page->table->Details->View
Issue 2. Same issue reproes on Traces page->Tree control- user can adjust the width of the tree control by dragging.**

### Expected Results:
There should be functionality provided to adjust the height of the "Details View" section using single pointer.

### User Impact:
Functionality provided only with dragging will create difficulty for users using input devices like a trackball, head pointer, eye-gaze system to perform the operation.

### Attachment:
[3365_A11y_ NetAspireDashboard_Resources_Detailsview_DraggingMovement.webm](https://github.com/dotnet/aspire/assets/90901519/62e97ef5-9e41-4bff-bbab-292651b509cc)

![3365_A11y_ NetAspireDashboard_Metrics_TreeControl_DraggingMovement](https://github.com/dotnet/aspire/assets/90901519/cb44d2e9-e0dd-4201-aff6-75d28fb90234)
